static/js/compose.js

```diff
@@ -704,6 +704,14 @@ export function initialize() {
         compose_actions.update_placeholder_text();
     });
 
+    $("body").on("click", ".formatting_button", (e) => {
+        const $elt = $(e.target);
+        const textarea = $elt.closest("form").find("textarea");
+        const format_type = $elt.data("format-type");
```

Comment:
We should use `.attr("data-format-type"`).  The jQuery `.data()` function is evil, both in that it's not great for grepping, and more importantly because if you store values using it, they are only represented in the jQuery world, and not the DOM, so they don't inter-operate with using the attribute.